Week-1/Session-04_Advanced_HTML/Assignments/birthday/Style.css

```diff
@@ -0,0 +1,16 @@
+form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+form div {
+    display: flex;
+    width: 25%;
```

Comment:
Is this working?

As I can see you dont have any perticalar width property for the form, % will only work if you have specified the width to the parent element